tests/braille-specs/Cop-Eg.yaml

```diff
@@ -0,0 +1,26 @@
+# Yaml Test For Coptic Computer Braille Table 
+# Copyright (C) 2020 By Ibraam Nasif, St. Didimous Group  Coptic Bishopric of Youth - Egypt
+
+  display: unicode.dis
+  table: Cop-eg-comp8.utb
+  flags: {testmode: forward}
+  tests:
+    
+    # Coptic
+  - ['ⲁⲃⲅⲇⲉⲋⲍⲏⲑⲓⲕⲗⲙⲛⲝⲟⲡⲣⲥⲧⲩⲫⲭⲯⲱϣϥϧϩϫϭϯ', '⠁⠃⠛⠙⠑⠉⠵⠱⠹⠊⠅⠇⠍⠝⠭⠕⠏⠗⠎⠞⠥⠋⠯⠽⠺⠩⠧⠣⠓⠚⠬⠫']
+  - ['ⲀⲂⲄⲆⲈⲊⲌⲎⲐⲒⲔⲖⲘⲚⲜⲞⲠⲢⲤⲦⲨⲪⲬⲮⲰϢϤϦϨϪϬϮ', '⡁⡃⡛⡙⡑⡉⡵⡱⡹⡊⡅⡇⡍⡝⡭⡕⡏⡗⡎⡞⡥⡋⡯⡽⡺⡩⡧⡣⡓⡚⡬⡫']
+  - ['̀̄̅', '⠟⠰⠰']
+  
+    # swap diacritics 
+  - ['ⲫ̀ⲓⲱⲧ Ⲡ̀ϭⲟⲓⲥ Ⲛⲏⲉ̅ⲑ̅ⲩ̅', '⠟⠋⠊⠺⠞⠀⠟⡏⠬⠕⠊⠎⠀⡝⠱⠰⠑⠰⠹⠰⠥' {xfail: true  Swap section}]
```

Comment:
@IbraamNasif Is this test really expected to fail? You seem to have some rules in the table to take care of this.